### ISSUE TITLE
Search string in results is white on white

### ISSUE THREAD

**Issue by rzw** [Zettlr/Zettlr#5315]
### Description

White font on white background seems strange. With inspect element and CSS I was able to fix (but it shouldn't be necessary):

```
.search-result-highlight {
color:orangered !important;
}
```

![Screenshot (1884)](https://github.com/user-attachments/assets/42767fbe-ce7b-4c59-ac54-ca864159dd7b)

### Reproducing

Install, open, do full-text search

### Zettlr version

3.2.0

### Installation Method

From the Website or GitHub

### Your Platform

- [X] Windows
- [ ] macOS
- [ ] Linux

### Architecture

- [X] Intel 64bit
- [ ] ARM 64bit (includes Apple Silicon)

### Operating System Version

Windows 10

**Comment by nathanlesage** [comment]
The specific color you refer to uses the system's accent color, i.e. the accent color you selected in the Windows settings. I notice that your title bar (which uses the accent color as well) seems very light. This, or a wrong Custom CSS entry might be the culprit.

Please try these two things:

1. Try to temporarily switch to a dark accent color in the Windows settings (maybe restart the app). Does this improve the visuals?
2. Check your Custom CSS rules whether some of them might interfere — for that, simply comment out all your Custom CSS so that you only get the built-in styles. Does this improve the visuals?

Afterwards, we can revise any next steps.

**Comment by rzw** [comment]
Thanks for your quick response. Yes, my windows settings were off. Usually I don't touch. But recently out of nowhere I had a bamboo jungle as background and green biased color settings. I must have got it wrong while going back to my preferred settings.

Nonetheless for the record:
I had tried No.1. And regarding No.2: You are right. At the time of the screenshot I had custom CSS settings. (Sort of stabbing in the dark before I discovered debug mode and found "inspect element".) But they hadn't changed a thing regarding the left global search bar compared with the initially empty custom CSS.

Thanks for your help.
